src/NuGetGallery.Core/GitHub/RepositoryInformation.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+
+namespace NuGetGallery.GitHub
+{
+    public class RepositoryInformation
+    {
+        public RepositoryInformation()
+        { }
+
+        public RepositoryInformation(string owner, string repoName, string cloneUrl, int starCount, List<string> dependencies)
```

Comment:
Consider breaking this up for readability. Something like
```suggestion
        public RepositoryInformation(
            string owner,
            string repoName,
            ...)
```

Reply:
Should `dependencies` be an `IReadOnlyList<string>`? If so, please update line 40 too